src/main.sh

```diff
@@ -52,10 +52,10 @@ declare -r startdir="$(pwd -P)"
 declare -r MAKEDEB_VERSION='{MAKEDEB_VERSION}'
 declare -r MAKEDEB_RELEASE='{MAKEDEB_RELEASE}'
 declare -r MAKEDEB_INSTALLATION_SOURCE='{MAKEDEB_INSTALLATION_SOURCE}'
-declare -r MAKEDEB_DPKG_ARCHITECTURE="$(dpkg --print-architecture)"
 declare -r MAKEDEB_DISTRO_CODENAME="$(lsb_release -cs)"
```

Comment:
Could you make this `MAKEDEB_DISTRO_CODENAME` line function the same way too? Could the `declare -r` declarations also be kept for both?

Reply:
`makedeb --lint` fails with it?

Reply:
@pirate486743186 any update?

Reply:
> but if you declare it inside the PKGBUID, the variable is already read only and complains....

Is that the only place the error happens? The main purpose of making it `declare -r` was exactly so that it couldn't be changed in a PKGBUILD, though I'd be open to changing it if you have any reason against it. For the time being I'd definitely probably want it to be `declare -r` though

Reply:
Looks good to me, thanks! I'll get it merged right after the CI runs :)